match/lib/match/options.rb

```diff
@@ -292,6 +292,11 @@ def self.available_options
                                      description: "Renew the provisioning profiles if the certificate count on the developer portal has changed. Works only for the 'development' provisioning profile type. Requires 'include_all_certificates' option to be 'true'",
                                      type: Boolean,
                                      default_value: false),
+        FastlaneCore::ConfigItem.new(key: :renew_expired_certs,
+                                     env_name: "MATCH_RENEW_EXPIRED_CERTS",
+                                     description: "Automatically renew expired certificates. Notice: to renew developer_id and developer_id_installer certificates you must login with Account Holder account",
```

Comment:
I reviewed this and the implementation and it seems users will need to authenticate via username + password + 2FA if they wish to renew those 2 certificates. Perhaps we can make this more clear? Because via App Store Connect API Key the highest role is Admin only 😬 which is what most people are using nowadays probably, due to so many challenges around regular auth…

How about…

```suggestion
                                     description: "Automatically renew expired certificates. Note: to renew `developer_id` and `developer_id_installer` certificates you must log in with the Account Holder account by using username and password; App Store Connect API key doesn't work in this case",
```

Reply:
Thanks, updated the description.